README.md

```diff
@@ -88,16 +88,16 @@ export VISIER_APIKEY=the-api-key-issued-by-visier
 export VISIER_CLIENT_ID=
 ```
 
-Source this environment in and provide the password when prompted:
+Next, source this environment in and provide the password when prompted.
 ```sh
 $ source .env
 ```
 
 ## Jupyter Notebooks
 Jupyter notebooks and lab are well-suited to run Visier connector code. However, some users may not find OS-level variables ideal. As of version `0.9.9`, the Visier Python connector supports [dotenv](https://pypi.org/project/python-dotenv/) to facilitate a more dynamic switching of Visier authentication parameters. If the file is called `.env`, the Python package `dotenv` attempts to load the file. If the file has a different name, you must provide that file name when loading the environment with `dotenv`.
 
-### Jupyter Basic Authentication Example
-Basic Authentication is the most practical means of authenticating against Visier for Jupyter notebooks.
+### Jupyter Password Authentication Example
```

Comment:
I don't understand this. Basic Authentication is the official term for username/password auth. 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#authentication_schemes

Reply:
In our api-samples repo, we refer to authorization_code and password auth, which I assumed are the exact same as these 2 options. I changed it here to be more consistent with the other repo: https://github.com/visier/api-samples/tree/main/authentication

Reply:
There's a difference between OAuth grant types (authorization code versus password) and Basic authentication in HTTP.

What do you think about this:
* For anything related to OAuth we're careful to refer to grant types (as opposed to authentication type)
* For legacy ASID authentication, we call it Basic in the sense it's only requiring username and password (even though it does not use the HTTP Basic authorization header. Sigh...)

Reply:
I see! Yup, that distinction sounds good to me.